docs/workflows/syntax/basic-syntax.mdx

```diff
@@ -126,5 +126,5 @@ on-failure:
       config: " {{ providers.slack-demo }} "
 ```
 
-On-failure is actually a condtionless `Action` used to notify in case the alert failed with an exception.
+On-failure is actually an unconditional `Action` used to notify in case the alert failed with an exception.
```

Comment:
The meaning was an action with no condition. Unconditional sounds weird.

Reply:
@talboren Yeah, I got that meaning. But `conditionless` is not a correct word. If `unconditional` sounds weird, then we can simply keep it as `without any condition` or `with no condition`. We can also use `an unconditioned Action`. Please let me know what you think about it.

Reply:
with no condition sounds best.

Reply:
Thanks. Just so you know – all suggestions are applied.